docs/source/examples/vision/guides/formatters_validating.ipynb

```diff
@@ -0,0 +1,464 @@
+{
```

Comment:
<p>as they're meant to -&gt; as intended</p><p> </p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/deepchecks/deepchecks/pull/875/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='0'/>